src/lib/tls/msg_cert_verify.cpp

```diff
@@ -62,11 +62,11 @@ std::vector<uint8_t> Certificate_Verify::serialize() const
    {
    std::vector<uint8_t> buf;
 
-   if(m_scheme != Signature_Scheme::NONE)
+   // TODO: both for TLS 1.2 and 1.3 this should never happen and the message would be invalid
+   if(m_scheme.is_set())
       {
-      const uint16_t scheme_code = static_cast<uint16_t>(m_scheme);
-      buf.push_back(get_byte<0>(scheme_code));
-      buf.push_back(get_byte<1>(scheme_code));
+      buf.push_back(get_byte<0>(m_scheme.wire_code()));
+      buf.push_back(get_byte<1>(m_scheme.wire_code()));
```

Comment:
This is a very minor point but perhaps easier to read with a variable to hold the result of `m_scheme.wire_code`